tripy/tripy/frontend/module/layernorm.py

```diff
@@ -92,9 +104,18 @@ def __call__(self, x: "tripy.Tensor") -> "tripy.Tensor":
             A tensor of the same shape as the input.
         """
         from tripy.frontend.trace.ops.reduce import mean, var
+        from tripy.frontend.shape import Shape
         from tripy.frontend.trace.ops.unary_elementwise import rsqrt
+        from tripy.common.exception import raise_error
+
+        # The mean and the variance are computed over the last D dimensions
+        D = len(self.normalized_shape)
+
+        if x.shape[-D:] != Shape(self.normalized_shape):
+            raise_error(f"The input's last {D} dimensions must have a shape of {self.normalized_shape} and received {x.shape[-D:].as_tensor().data()}")
```

Comment:
```suggestion
            raise_error(f"The input's last {D} dimensions must have a shape of {self.normalized_shape} but received {x.shape[-D:].as_tensor().data()}")
```

Reply:
Can we make the summary line something like "Unexpected input shape" and move this to the details section? Also, do we need to change the shape tensor to a regular tensor here?